src/features/newCommentForm.ts

```diff
@@ -0,0 +1,54 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+interface NewFormState {
+  name: string;
+  email: string;
+  body: string;
+}
+
+const initialState: NewFormState = {
+  name: '',
+  email: '',
+  body: '',
+};
+
+export const newFormSlice = createSlice({
+  name: 'newForm',
+  initialState,
+  reducers: {
+    setName: (state, action) => {
+      return {
+        ...state,
+        name: action.payload,
+      };
+    },
+    setEmail: (state, action) => {
+      return {
+        ...state,
+        email: action.payload,
+      };
+    },
+    setBody: (state, action) => {
+      return {
+        ...state,
+        body: action.payload,
+      };
+    },
+    clearTwoFiledForm: state => {
```

Comment:
There is a typo in the reducer name `clearTwoFiledForm`. It should be `clearTwoFieldForm` to maintain consistency and clarity in naming conventions.